docs/reference/time_series/api.md

```diff
@@ -0,0 +1,324 @@
+# TimeSeries LLM Forecast API Documentation
+
+## Overview
+
+The TimeSeries LLM Forecast API enables users to perform advanced time series forecasting using transformer-based models. This API supports both univariate and multivariate time series, provides quantile forecasting, and is designed to be extensible for future enhancements. The API is compatible with standard RESTful practices and allows for flexible options and metadata handling, making it suitable for a wide range of time series applications across industries.
```

Comment:
Is it worth adding some small introduction or a link about what what is forecasting and simple use cases for people that might be reading about it for the first time?